pom.xml

```diff
@@ -31,16 +31,14 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-batch</artifactId>
     </dependency>
-
-	<dependency>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-test</artifactId>
-	</dependency>
-
-	<dependency>
-		<groupId>org.springframework.batch</groupId>
-		<artifactId>spring-batch-test</artifactId>
-	</dependency>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
```

Comment:
Thank you for getting rid of the tabs here. Maybe standardize to 2 space indenting as well?